Examples/Tests/photon_photon_collisions/inputs_two_photons_3d

```diff
@@ -0,0 +1,137 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 100
+amr.n_cell = 16 16 16
+amr.max_grid_size = 16
+amr.blocking_factor = 4
```

Comment:
Is there a reason to keep the blocking factor so low? If you want to test if the code works with multiple boxes you could set both `max_grid_size` and `blocking_factor` to 8